internal/backoff/exponential.go

```diff
@@ -35,8 +35,7 @@ type ExponentialOption func(*exponentialOptions)
 // exponentialOptions are the configuration options for an exponential backoff
 type exponentialOptions struct {
 	base, min, max time.Duration
```

Comment:
Under what circumstances would I want a non-zero minimum? As long as we're jittering, isn't zero safe for the server and fastest for the client?